icu4j/main/core/src/test/java/com/ibm/icu/dev/test/util/ICUServiceTestSample.java

```diff
@@ -60,13 +59,11 @@ public void helloServiceChanged() {
         }
 
         private void display() {
-            Map names = HelloService.getDisplayNames(ULocale.US);
+            Map<String, String> names = HelloService.getDisplayNames(ULocale.US);
             System.out.println("displaying " + names.size() + " names.");
-            Iterator iter = names.entrySet().iterator();
-            while (iter.hasNext()) {
-                Entry entry = (Entry)iter.next();
-                String displayName = (String)entry.getKey();
-                HelloService service = HelloService.get((String)entry.getValue());
+            for (Entry<String, String> entry : names.entrySet()) {
```

Comment:
ditto optional: not import Map.Entry (I know, pre-existing)

Reply:
Done